File: bookstore/orders/src/Order.js
const axios = require('axios').default;
// class Order {
//   constructor(bookId, quantityOrdered) {
//     this.orderId = Math.floor((Math.random() * 100) + 1);
//     this.bookId = bookId;
//     this.quantityOrdered = quantityOrdered;
//     this.status = null;


    
//   }
//   getOrderId() {
//     return this.orderId;
//   }
//   getStatus(){
//     return this.status;
//   }
Order = {
  test: async function createOrder (bookId){
    return axios.get('http://localhost:4568/books/' + bookId)
  }
};
//   createAudioFileAsync(audioSettings, successCallback, failureCallback);
// }

exports.Order = Order;

File: orders/src/main/java/com/example/orders/Order.java
package com.example.orders;
import java.net.*;

public class Order {
    private long orderId;
    private String bookId;
    private int quantityOrdered;
    String status;

    public Order() {
        this.orderId = 0;
        this.bookId = "";
        this.quantityOrdered = 0;
    }
    public Order(String bookId, int quantityOrdered){
        this.orderId = 0;
        this.bookId = bookId;
        this.quantityOrdered = quantityOrdered;
    }

    public long getOrderId(){
        return this.orderId;
    }
    public void setOrderId(long orderId){
        this.orderId = orderId;
    }
    public String getBookId(){
        return this.bookId;
    }
    public void setBookId(String bookId){
        this.bookId = bookId;
    }
    public int getQuantityOrdered(){
        return this.quantityOrdered;
    }
    public void setQuantityOrdered(int quantityOrdered){
        this.quantityOrdered = quantityOrdered;
    }

}
